src/coreComponents/mesh/WellElementSubRegion.cpp

```diff
@@ -137,6 +138,33 @@ void collectElementNodes( CellElementSubRegion const & subRegion,
   }
 }
 
+template< typename SUBREGION_TYPE >
+bool isPointInsideElement( SUBREGION_TYPE const & GEOS_UNUSED_PARAM( subRegion ),
+                           arrayView2d< real64 const, nodes::REFERENCE_POSITION_USD > const & GEOS_UNUSED_PARAM( referencePosition ),
+                           localIndex const & GEOS_UNUSED_PARAM( eiLocal ),
+                           ArrayOfArraysView< localIndex const > const & GEOS_UNUSED_PARAM( facesToNodes ),
+                           real64 const (&GEOS_UNUSED_PARAM( elemCenter ))[3],
+                           real64 const (&GEOS_UNUSED_PARAM( location ))[3] )
+{
+  // only CellElementSubRegion is currently supported
```

Comment:
I may be wrong, but this code should not run if not supported.
```suggestion
  static_assert( false, "only CellElementSubRegion is currently supported" );
```

Reply:
i think it will start crashing if i add that assert
the needed functions are implemented in https://github.com/GEOS-DEV/GEOS/pull/3359
hope we can merge both soon